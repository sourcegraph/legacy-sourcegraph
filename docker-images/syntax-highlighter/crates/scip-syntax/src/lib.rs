pub mod evaluate;
pub mod index;
pub mod io;
pub mod progress;
pub mod scip_strict;
