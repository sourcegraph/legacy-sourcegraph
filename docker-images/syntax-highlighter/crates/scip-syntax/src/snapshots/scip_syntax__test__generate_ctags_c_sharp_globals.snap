---
source: crates/scip-syntax/src/lib.rs
expression: "String::from_utf8_lossy(buf_writer.buffer())"
---
{"_type":"tag","name":"Longer.Namespace","path":"globals.cs","language":"c_sharp","line":5,"kind":"namespace","scope":null}
{"_type":"tag","name":"Class","path":"globals.cs","language":"c_sharp","line":7,"kind":"type","scope":"Longer.Namespace"}
{"_type":"tag","name":"Swag","path":"globals.cs","language":"c_sharp","line":39,"kind":"type","scope":"Longer.Namespace.Class"}
{"_type":"tag","name":"Pants","path":"globals.cs","language":"c_sharp","line":44,"kind":"variable","scope":"Longer.Namespace.Class.Swag"}
{"_type":"tag","name":"Socks","path":"globals.cs","language":"c_sharp","line":43,"kind":"variable","scope":"Longer.Namespace.Class.Swag"}
{"_type":"tag","name":"Sweater","path":"globals.cs","language":"c_sharp","line":42,"kind":"variable","scope":"Longer.Namespace.Class.Swag"}
{"_type":"tag","name":"Shirt","path":"globals.cs","language":"c_sharp","line":41,"kind":"variable","scope":"Longer.Namespace.Class.Swag"}
{"_type":"tag","name":"ZigFeatureSet","path":"globals.cs","language":"c_sharp","line":49,"kind":"type","scope":"Longer.Namespace.Class"}
{"_type":"tag","name":"FullPackage","path":"globals.cs","language":"c_sharp","line":57,"kind":"variable","scope":"Longer.Namespace.Class.ZigFeatureSet"}
{"_type":"tag","name":"CoolCommunity","path":"globals.cs","language":"c_sharp","line":56,"kind":"variable","scope":"Longer.Namespace.Class.ZigFeatureSet"}
{"_type":"tag","name":"BuildSystem","path":"globals.cs","language":"c_sharp","line":55,"kind":"variable","scope":"Longer.Namespace.Class.ZigFeatureSet"}
{"_type":"tag","name":"Comptime","path":"globals.cs","language":"c_sharp","line":54,"kind":"variable","scope":"Longer.Namespace.Class.ZigFeatureSet"}
{"_type":"tag","name":"Errors","path":"globals.cs","language":"c_sharp","line":53,"kind":"variable","scope":"Longer.Namespace.Class.ZigFeatureSet"}
{"_type":"tag","name":"None","path":"globals.cs","language":"c_sharp","line":52,"kind":"variable","scope":"Longer.Namespace.Class.ZigFeatureSet"}
{"_type":"tag","name":"Syntax","path":"globals.cs","language":"c_sharp","line":60,"kind":"method","scope":"Longer.Namespace.Class"}
{"_type":"tag","name":"SourcegraphSwag","path":"globals.cs","language":"c_sharp","line":47,"kind":"variable","scope":"Longer.Namespace.Class"}
{"_type":"tag","name":"LambdaFunction","path":"globals.cs","language":"c_sharp","line":37,"kind":"variable","scope":"Longer.Namespace.Class"}
{"_type":"tag","name":"PublicImplicitGetterSetter","path":"globals.cs","language":"c_sharp","line":35,"kind":"variable","scope":"Longer.Namespace.Class"}
{"_type":"tag","name":"_speed","path":"globals.cs","language":"c_sharp","line":34,"kind":"variable","scope":"Longer.Namespace.Class"}
{"_type":"tag","name":"ImplicitGetterPrivateSetter","path":"globals.cs","language":"c_sharp","line":28,"kind":"variable","scope":"Longer.Namespace.Class"}
{"_type":"tag","name":"ImplicitGetterSetter","path":"globals.cs","language":"c_sharp","line":22,"kind":"variable","scope":"Longer.Namespace.Class"}
{"_type":"tag","name":"_val","path":"globals.cs","language":"c_sharp","line":20,"kind":"variable","scope":"Longer.Namespace.Class"}
{"_type":"tag","name":"ExplicitGetterSetter","path":"globals.cs","language":"c_sharp","line":9,"kind":"variable","scope":"Longer.Namespace.Class"}
{"_type":"tag","name":"SurprisinglyValid","path":"globals.cs","language":"c_sharp","line":3,"kind":"method","scope":null}
{"_type":"tag","name":"System.Collections.Generic","path":"globals.cs","language":"c_sharp","line":1,"kind":"type","scope":null}

